assets/airports/panc.json

```diff
@@ -0,0 +1,2164 @@
+            },
+            "body": [["PTERS", "A160+"], ["SKENS", "A130+|A150-"], ["MMMAT", "A100+"], ["BBELL", "A80+"], ["DDOUG", "A80"], "LMPEE", "#327"],
+            "rwy": {
+                "15": []
+            },
+            "draw": [
+                ["GKN", "PTERS", "SKENS", "MMMAT", "BBELL", "DDOUG", "LMPEE"],
+                ["JOH", "DENSN", "PTERS"],
+                ["REMBY", "PTERS"],
+                ["TIEKL", "PTERS"],
+                ["TMSON", "PTERS"],
+                ["MDO", "DENSN"]
+            ]
+        },
+        "TAGER6": {
+            "icao": "TAGER6",
+            "name": "Tagger Six",
+            "entryPoints": {
+                "ENN": ["ENN", "TKA"],
+                "FFITZ": ["FFITZ", "ROHHN"],
+                "GAL": ["GAL", "AKMAX", "ROHHN"],
+                "MCG": ["MCG", "FFITZ", "ROHHN"],
+                "TKA": ["TKA"]
+            },
+            "body": ["TAGER", "TED"],
+            "rwy": {
+                "7L": [],
+                "7R": [],
+                "15": [],
+                "25L": [],
+                "25R": [],
+                "33": []
+            },
+            "draw": [
+                ["MCG", "FFITZ", "ROHHN", "TAGER"],
+                ["ENN", "TKA", "TAGER", "TED"],
+                ["GAL", "AKMAX", "ROHHN"]
+            ]
+        },
+        "WITTI3": {
+            "icao": "WITTI3",
+            "name": "Witty Three",
+            "entryPoints": {
+                "JOH": ["JOH"],
+                "MDO": [["MDO", "A260+"]],
+                "REMBY": [["REMBY", "A260+"]],
+                "TIEKL": [["TIEKL", "A260+"], "OWXUP"],
+                "TMSON": [["TMSON", "A260+"]]
+            },
+            "body": [["WITTI", "A160+"], ["GAYLL", "A120+|A150-"], ["SLOOW", "A80+"], ["SILTY", "A54+"], ["ANDDI", "A50|S210"], "#251"],
+            "rwy": {
+                "7L": [],
+                "7R": []
+            },
+            "draw": [
+                ["JOH", "WITTI", "GAYLL", "SLOOW", "SILTY", "ANDDI"],
+                ["TIEKL", "OWXUP", "WITTI"],
+                ["REMBY", "WITTI"],
+                ["TMSON", "WITTI"],
+                ["MDO", "WITTI"]
+            ]
+        },
+        "YESKA6": {
+            "icao": "YESKA6",
+            "name": "Yeska Six",
+            "entryPoints": {
+                "JOH": ["JOH", "SOBGE"],
+                "MDO": ["MDO", "SOBGE"]
+            },
+            "body": ["YESKA", "TED"],
+            "rwy": {
+                "7L": [],
+                "7R": [],
+                "15": [],
+                "25L": [],
+                "25R": [],
+                "33": []
+            },
+            "draw": [
+                ["JOH", "SOBGE", "YESKA", "TED"],
+                ["MDO", "SOBGE"]
+            ]
+        }
+    },
+    "spawnPatterns": [
+        {
+            "origin": "panc",
+            "destination": "",
+            "category": "departure",
+            "route": "PANC.ANC8.BGQ..FAI",
```

Comment:
`BGQ` is not an exit of the ANC8, so this is causing some errors. Perhaps we could do `PANC.ANC8.TED..BGQ..FAI`? Not like they would ever actually fly like that, but it wouldn't surprise me in the least if they were filed like that, and then just cleared direct BGQ upon contacting departure.

Reply:
Yeah this was meant to be `PANC.ANC8.TED..BGQ..FAI`

Reply:
done 👍